src/ontology/edits/oeo-physical.omn

```diff
@@ -8097,8 +8097,19 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/608",
     
 Class: OEO_00050016
 
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared"^^xsd:string,
```

Comment:
```suggestion
        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
```
This might be good to avoid confusion.